src/internal/kopia/base_finder.go

```diff
@@ -13,8 +13,40 @@ import (
 	"github.com/alcionai/corso/src/internal/operations/inject"
 	"github.com/alcionai/corso/src/pkg/backup"
 	"github.com/alcionai/corso/src/pkg/logger"
+	"github.com/alcionai/corso/src/pkg/path"
 )
 
+const (
```

Comment:
changes in this file are just pulling in things that were in the snapshot manager